Caltech-IPAC/firefly
# DM-16777: Cleanup, updates, and documentation of who we hand meta data

Cleanup, updates, and documentation of meta data

   - documented all meta in MetaConst.js and MetaConst.java
   - removed or deprecated unnecessary or redundent meta data
   - cleaned up definition of catalog
   - moved more logic into VOAnalyzer
   - refactors some search processors that use meta data
   - Added more testing

_To Test:_

https://irsawebdev9.ipac.caltech.edu/dm-16777-metadata/firefly/

Everything should work the same as before

  - Search a catalog
  - Search a votable
  - Search a wise image (under test searches)
